test-data/run.test

```diff
@@ -279,3 +279,13 @@ assert f(None) is None
 assert f(a) is a
 assert g(None) == 1
 assert g(a) == 2
+
+[case testUnicodeLiteral]
```

Comment:
Test other special characters in this test case, such as `\n` and double quotes. I assume that null characters won't work -- can you create an issue about supporting them (not important now, but would be nice eventually)?

Reply:
Supporting nulls wasn't too hard